clients/include/norm.hpp

```diff
@@ -239,14 +239,15 @@ double norm_check_general(char           norm_type,
 }
 
 /* ============== Norm Check for batched case ============= */
+
 template <typename T, typename T_hpa>
-double norm_check_general(char               norm_type,
-                          rocblas_int        M,
-                          rocblas_int        N,
-                          rocblas_int        lda,
-                          host_vector<T_hpa> hCPU[],
-                          host_vector<T>     hGPU[],
```

Comment:
Are you sure you replaced all the old style host_vector arrays?  I guess CI testing will tell.